### PR TITLE
[fix] Fixed project configuration for RADIUS module

### PR DESCRIPTION
**Pull request by pandafy**
## Checklist

- [x] I have read the [OpenWISP Contributing Guidelines](http://openwisp.io/docs/developer/contributing.html).
- [x] I have manually tested the changes proposed in this pull request.
- [x] I have written new test cases for new code and/or updated existing tests for changes to existing code.
- [ ] I have updated the documentation.

## Description of Changes

Fixed bug with RADIUS API for user registration.

The bug was reported in https://github.com/orgs/openwisp/discussions/921

**TODOS**
- [x]  Add automated test

### CODE REVIEW

**Review by nemesifier — COMMENTED** [review]
I confirm the fix works! I think just 1 test will be enough, make sure it fails without the fix please @pandafy :pray:

File: tests/runtests.py [hunk 1]
```diff
@@ -419,6 +420,27 @@ def _test_celery_task_registered(container_name):
         with self.subTest('Test celery_monitoring container'):
             _test_celery_task_registered('celery_monitoring')
 
+    def test_radius_user_registration(self):
+        """Ensure users can register using the RADIUS API."""
+        url = f"{self.config['api_url']}/api/v1/radius/" "organization/default/account/"
```
Comment (nemesifier) [inline tests/runtests.py:425]:
```suggestion
        url = f'{self.config["api_url"]}/api/v1/radius/organization/default/account/'
```

**Review by nemesifier — APPROVED** [review]
@pandafy LGTM! Let me know if you agree with my latest change.

**Review by pandafy — COMMENTED** [review]
LGTM!

**Comment by 3pleo** [comment]
Thanks.
This has been tested. But following similar error pops up when "Confirm Email" link sent is clicked:

_NoReverseMatch at /accounts/confirm-email/NTg:1sqSLG:FeIEip6aYPX77iN8vZ99FWKa5XWWvmRUlD2TxOK3qlY/
'admin' is not a registered namespace_

find the full trace attached. 
[confirm_email_error.txt](https://github.com/user-attachments/files/17027543/confirm_email_error.txt)

Regards,

**Comment by nemesifier** [comment]
> Thanks. This has been tested. But following similar error pops up when "Confirm Email" link sent is clicked:
> 
> _NoReverseMatch at /accounts/confirm-email/NTg:1sqSLG:FeIEip6aYPX77iN8vZ99FWKa5XWWvmRUlD2TxOK3qlY/ 'admin' is not a registered namespace_
> 
> find the full trace attached. [confirm_email_error.txt](https://github.com/user-attachments/files/17027543/confirm_email_error.txt)
> 
> Regards

@3pleo please create a new issue for that.

**Pull request merged**